sr_blockly_blocks/generators/python/scripts/initialise.py

```diff
@@ -9,6 +9,8 @@
     hand_commander = SrHandCommander(name=side + "_hand")
 elif dropdown_commander == 'robot_commander':
     robot_commander = SrRobotCommander(name=side + "_arm_and_hand")
+elif dropdown_commander == 'hand_h_and_arm':
+    robot_commander = SrRobotCommander ("right_arm_and_hand", prefix="H1_")
```

Comment:
Could do but it would be misleading I think... The commander only works with one hand_h + arm. selecting left would never be correct and would break things. seems misleading to put something wrong just to be consistent.